packages/experimental/ExperimentalIcon/src/__tests__/__snapshots__/Icon.test.tsx.snap

```diff
@@ -0,0 +1,15 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Icon component tests SVG Icon 1`] = `
+<Text
+  style={
+    Object {
+      "color": "#f07",
+      "fontFamily": "Arial",
+      "fontSize": undefined,
```

Comment:
Not blocking since I'm guessing this will be fleshed out in the future, but it seems off that the fontSize is undefined